FinnUI.podspec

```diff
@@ -0,0 +1,39 @@
+Pod::Spec.new do |s|
+  s.name         = 'FinnUI'
+  s.version      = '1.0.0'
```

Comment:
> What I did to separate between FinniversKit and FinnUI releases is that the FinnUI release tag should be prefixed with finnui-, e.g. finnui-1.0.0.

Shouldn't this be equal to the tag we're checking out, i.e. `finnui-1.0.0`? Or doesn't Cocoapods look at this string?

Reply:
No, as far as I understand it is the actual version number of the library, but it is used to specify tag a little later on:
`s.source        = { :git => "https://github.com/finn-no/FinniversKit.git", :tag => "finnui-#{s.version}" }`

Reply:
Ah, I see. Ok, let's try it out 👍